src/bluefinClient.ts

```diff
@@ -251,9 +248,9 @@ export class BluefinClient {
    * @param keypair key pair for the account to be used for placing orders
    */
   initializeWithKeyPair = async (keypair: Keypair): Promise<void> => {
-    this.signer = new RawSigner(keypair, this.provider);
-    this.walletAddress = await this.signer.getAddress();
-    this.initOrderSigner(keypair);
+    this.signer = keypair;
+    this.walletAddress = await this.signer.toSuiAddress();
+    this.initOrderSigner(keypair as Keypair);
```

Comment:
Isn't the casting here redundant given the type this function expects is the same?